starter-code/french-onion-soup-starter-code/index.html

```diff
@@ -1,43 +1,77 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <title>French Onion Soup Recipe</title>
-  <!-- Google fonts gives us access to the Open Sans Font -->
-  <!-- Google fonts is also awesome! Make sure to check it out -->
-  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,600" rel="stylesheet">
-</head>
-<body>
-  French Onion Soup
-
-  We have been trying French onion soup in restaurants for years and my family
-and friends agree none can compare to my recipe for taste and simplicity of
-preparation.
 
-Ingredients
-1/2 cup unsalted butter
-2 tablespoons olive oil
-4 cups sliced onions
-4 (10.5 ounce) cans beef broth
-2 tablespoons dry sherry (optional)
-1 teaspoon dried thyme
-salt and pepper to taste
-4 slices French bread
-4 slices provolone cheese
-2 slices Swiss cheese, diced
-1/4 cup grated Parmesan cheese
+<head>
 
-Directions
-Melt butter with olive oil in an 8 quart stock pot on medium heat. Add onions and
-continually stir until tender and translucent. Do not brown the onions.
+  <meta charset="UTF-8">
+  <meta name ="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>French Onion Soup</title>
+  <link rel="stylesheet" href ="./css/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,600" rel="stylesheet">
 
-Add beef broth, sherry and thyme. Season with salt and pepper, and simmer for 30
-minutes.
+</head>
 
-Heat the oven broiler.
+<body>
 
-Ladle soup into oven safe serving bowls and place one slice of bread on top of each
-(bread may be broken into pieces if you prefer). Layer each slice of bread with a slice of
-provolone, 1/2 slice diced Swiss and 1 tablespoon Parmesan cheese. Place bowls on
-cookie sheet and broil in the preheated oven until cheese bubbles and browns slightly.
+  <header>
+    <h1>French Onion Soup</h1>
```

Comment:
you could add a class here 

```
 <header class="soup">
    <h1>French Onion Soup</h1>
</header>
```

which would allow you to add all css into one file